readme.md

```diff
@@ -252,6 +252,14 @@ Once `less-loader` is enabled, you can import `.less` file in your project.
 
 `.module.less` will use CSS Modules.
 
+> if you use TypeScript (npm init react-app my-app --typescript) with CSS Modules, you should edit `react-app-env.d.ts`.
+> ```
+> declare module '*.module.less' {
+>   const classes: { [key: string]: string };
+>   export default classes;
+> }
+> ```
```

Comment:
There should be no need to prefix all these lines with '>'. The '>' prefix makes this render as a quote.

Reply:
I also think so.